dts/bindings/ethernet/nxp,enet1g.yaml

```diff
@@ -13,3 +13,7 @@ properties:
 
   clocks:
     required: true
+
+  rdc:
+    type: int
+    description: Set the RDC permission for this peripheral
```

Comment:
please provide the meaning of the abbreviation in the description at least

Reply:
Fixed, thanks.